scripts/apidoc.go

```diff
@@ -0,0 +1,408 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/kr/text"
+
+	"github.com/hashicorp/vault/sdk/framework"
+)
+
+const (
+	doNotEdit     = `<!-- This section is auto-generated and must be updated in the Go source file. -->`
+	maxLineLength = 100
+	hangingIndent = 2
+)
+
+// Don't use the default flag set since it will pick up all of the -test
+// flags that get pulled in when importing the logical packages.
+var (
+	flags      = flag.NewFlagSet("apidoc", flag.ExitOnError)
+	sourceFile = flags.String("f", "openapi.json", "OpenAPI reference file")
+	root       = flags.String("r", "website/source/api", "Root folder for documentation source")
+	verbose    = flags.Bool("v", false, "Verbose output")
+	analyze    = flags.Bool("a", false, "Analyze auto-generate coverage")
+)
+
+var typeDefaults = map[string]string{
+	"string":  `""`,
+	"integer": `0`,
+	"array":   `[]`,
+	"object":  `{}`,
+}
+
+func main() {
+	flags.Parse(os.Args[1:])
+
+	oas, err := loadOpenAPI(*sourceFile)
+	if err != nil {
+		fmt.Println(err)
+		return
```

Comment:
Alternatively, might consider `os.Exit(1)`. Not sure if it really matters.

Reply:
Good call, feels like it does matter to me - getting a proper exit code matters for the netlify build.

Reply:
Updated.